File: C/1009.c
#include <stdio.h>

double main() {
	char nome;
		double salario, vendas, calculo;
			scanf("%s", &nome);
				scanf("%lf%lf", &salario, &vendas);
					calculo = (salario + ((vendas * 15)/ 100));
				printf("TOTAL = R$ %.2lf\n", calculo);
			return 0;
}

File: Python/1009.py
from decimal import Decimal
nome = str(input(""))
salariofixo = float(input(""))
totaldevendas = float(input(""))
total = ((totaldevendas*15)/100)+salariofixo
total = Decimal("%.2f"%total)
print("TOTAL = R$ " + str(total))